README.md

```diff
@@ -1,24 +1,214 @@
-# README
+<a name="readme-top"></a>
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+<div align="center">
 
-Things you may want to cover:
+  <h3><b>Doctors Reservation Backend</b></h3>
 
-* Ruby version
+</div>
 
-* System dependencies
+<!-- TABLE OF CONTENTS -->
 
-* Configuration
+# 📗 Table of Contents
 
-* Database creation
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 Doctors Reservation Backend ](#-Doctors-Reservation-Backend-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Setup database](#setup-database)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
 
-* Database initialization
+<!-- PROJECT DESCRIPTION -->
 
-* How to run the test suite
+# 📖 Doctors Reservation Backend <a name="about-project"></a>
 
-* Services (job queues, cache servers, search engines, etc.)
+**Doctors Reservation Backend** is a Rails API that serves as database to the [Doctors Reservation Backend Frontend](https://github.com/Iqbal-Elham/Doctor-Reservation-frontend) project.
```

Comment:
The link to the frontend should have a name of `Doctors Reservation Frontend` not `Doctors Reservation Backend Frontend`